import-service/handlers/importFileParser.ts

```diff
@@ -0,0 +1,40 @@
+import * as AWS from 'aws-sdk';
+import { BUCKET, REGION } from '../constants/constants';
+import csv from 'csv-parser';
+
+export const importFileParser = (event, _context) => {
```

Comment:
I couldn't check this, but can you try to load file with some data and check if all of them properly logged in console of your fileParse lambda?  (and also check if uploaded filed is properly deleted)
It looks like you are not waiting for stream to finish. 

This criteria may not be met:

`+1 - At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder)`

Reply:
> I couldn't check this, but can you try to load file with some data and check if all of them properly logged in console of your fileParse lambda? (and also check if uploaded filed is properly deleted)
> It looks like you are not waiting for stream to finish.
> 
> This criteria may not be met:
> 
> `+1 - At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder)`

Hello
I checked this case and it works fine. The **importFileParser** is not **async** function (there is no **async** operator), so a lambda works until the event loop will be empty